kernel/configs/arm64_kata_kvm_4.19.x

```diff
@@ -1141,6 +1175,13 @@ CONFIG_OF_NET=y
 CONFIG_OF_RESERVED_MEM=y
 # CONFIG_OF_OVERLAY is not set
 # CONFIG_PARPORT is not set
+CONFIG_PNP=y
+CONFIG_PNP_DEBUG_MESSAGES=y
```

Comment:
not sure if you set this deliberately? Payoff of size vs debuggability...

Reply:
disable it. ;)